src/Simplify_Call.cpp

```diff
@@ -238,13 +256,24 @@ Expr Simplify::visit(const Call *op, ExprInfo *info) {
             return a | b;
         }
     } else if (op->is_intrinsic(Call::bitwise_not)) {
-        Expr a = mutate(op->args[0], nullptr);
+        ExprInfo a_info;
+        Expr a = mutate(op->args[0], &a_info);
```

Comment:
Why not use the bits_known mask in the same way here that you do for and/or? negated constants are constants

Reply:
I was, but then I changed it to not actually compute BitsKnown here.